Makefile

```diff
@@ -0,0 +1,23 @@
+all: backend frontend
+
+backend:
+	docker build . -f docker/Dockerfile_server
+	docker-compose up -d server
```

Comment:
1. Zaktualizujcie Docker Compose do wersji 2.X, wtedy komenda to `docker compose` (bez pauzy)
2. Polecam używanie `COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1`: https://www.docker.com/blog/faster-builds-in-compose-thanks-to-buildkit-support/, https://docs.docker.com/build/buildkit/
3. Polecam `--detach` zamiast `-d`, wtedy osoba słabiej znająca Dockera od razu wie, o co chodzi